django-stubs/db/models/fields/related.pyi

```diff
@@ -87,6 +94,9 @@ class RelatedField(FieldCacheMixin, Field[_ST, _GT]):
 
 class ForeignObject(RelatedField[_ST, _GT]):
```

Comment:
Looks like this class has lots of unsolved problems. Let's solve them!

1. Please, use `ClassVar` where needed: https://github.com/django/django/blob/a09082a9bec18f8e3ee8c10d473013ec67ffe93b/django/db/models/fields/related.py#L522-L530

Reply:
Thank you. As you suggested, I have handled all the relevant parts as class variables.